docs/manifest.json

```diff
@@ -1022,7 +1022,7 @@
 	{
 		"title": "ResizeTooltip",
 		"slug": "resize-tooltip",
-		"markdown_source": "../packages/components/src/resize-tooltip/README.md",
+		"markdown_source": "../packages/components/src/resizable-box/resize-tooltip/README.md",
```

Comment:
Should no longer be in the manifest, right?

Reply:
I'm... not sure. This was auto generated. There's a `README.md` inside the directory. I guess I can remove it? Ideally I'd like to preserve the docs though.

An alternative would be to inline comment some of the documentation (for now)

Reply:
Hmm... yeah, I see now. Well, I don’t have a preference. Maybe @nosolosw has a different take on how we should document components and let these tools work for us? Anyway, LGTM :)

Reply:
I took a peek at how the script that processes the handbook docs works. It seems that it looks for README files within the components folder ([see](https://github.com/WordPress/gutenberg/blob/master/docs/tool/manifest.js#L9)) and then updates the manifest used to expose them in the handbook (see [ResizableBox page](https://developer.wordpress.org/block-editor/components/resize-tooltip/)).

I'm unfamiliar with how we deal with subcomponents that aren't exposed via the `@wordpress/components` package. I had a quick look and this seems to be the only case? Anyway, it seems that we should either expose the resize-tooltip as a component proper or look at ways to remove its page from the handbook as it can't be used by external consumers of the API. For the latter, we could go with a quick hack (use a lowercased readme, use any other name for the file) or fix the issue (update the glob pattern to only match README files within the top-level component folder).